[2022-06-26 06:20:43] main.CRITICAL: Class Magento\Framework\App\Http\Interceptor does not exist {"exception":"[object] (ReflectionException(code: -1): Class Magento\\Framework\\App\\Http\\Interceptor does not exist at /var/www/html/magento2/vendor/magento/framework/Code/Reader/ClassReader.php:28)"} []
[2022-06-26 06:20:51] main.CRITICAL: Class Magento\Framework\App\Http\Interceptor does not exist {"exception":"[object] (ReflectionException(code: -1): Class Magento\\Framework\\App\\Http\\Interceptor does not exist at /var/www/html/magento2/vendor/magento/framework/Code/Reader/ClassReader.php:28)"} []
[2022-06-26 06:21:09] main.CRITICAL: Class Magento\Framework\App\Http\Interceptor does not exist {"exception":"[object] (ReflectionException(code: -1): Class Magento\\Framework\\App\\Http\\Interceptor does not exist at /var/www/html/magento2/vendor/magento/framework/Code/Reader/ClassReader.php:28)"} []
[2022-06-26 06:21:18] main.CRITICAL: Class Magento\Framework\App\Http\Interceptor does not exist {"exception":"[object] (ReflectionException(code: -1): Class Magento\\Framework\\App\\Http\\Interceptor does not exist at /var/www/html/magento2/vendor/magento/framework/Code/Reader/ClassReader.php:28)"} []
[2022-06-26 06:28:41] main.CRITICAL: SQLSTATE[HY000] [1698] Access denied for user 'root'@'localhost' {"exception":"[object] (Zend_Db_Adapter_Exception(code: 1698): SQLSTATE[HY000] [1698] Access denied for user 'root'@'localhost' at /var/www/html/magento2/vendor/magento/zendframework1/library/Zend/Db/Adapter/Pdo/Abstract.php:144, PDOException(code: 1698): SQLSTATE[HY000] [1698] Access denied for user 'root'@'localhost' at /var/www/html/magento2/vendor/magento/zendframework1/library/Zend/Db/Adapter/Pdo/Abstract.php:128)"} []
[2022-06-26 06:28:43] main.CRITICAL: SQLSTATE[HY000] [1698] Access denied for user 'root'@'localhost' {"exception":"[object] (Zend_Db_Adapter_Exception(code: 1698): SQLSTATE[HY000] [1698] Access denied for user 'root'@'localhost' at /var/www/html/magento2/vendor/magento/zendframework1/library/Zend/Db/Adapter/Pdo/Abstract.php:144, PDOException(code: 1698): SQLSTATE[HY000] [1698] Access denied for user 'root'@'localhost' at /var/www/html/magento2/vendor/magento/zendframework1/library/Zend/Db/Adapter/Pdo/Abstract.php:128)"} []
[2022-06-26 06:33:14] main.CRITICAL: SQLSTATE[HY000] [1698] Access denied for user 'root'@'localhost' {"exception":"[object] (Zend_Db_Adapter_Exception(code: 1698): SQLSTATE[HY000] [1698] Access denied for user 'root'@'localhost' at /var/www/html/magento2/vendor/magento/zendframework1/library/Zend/Db/Adapter/Pdo/Abstract.php:144, PDOException(code: 1698): SQLSTATE[HY000] [1698] Access denied for user 'root'@'localhost' at /var/www/html/magento2/vendor/magento/zendframework1/library/Zend/Db/Adapter/Pdo/Abstract.php:128)"} []
[2022-06-26 06:33:15] main.CRITICAL: SQLSTATE[HY000] [1698] Access denied for user 'root'@'localhost' {"exception":"[object] (Zend_Db_Adapter_Exception(code: 1698): SQLSTATE[HY000] [1698] Access denied for user 'root'@'localhost' at /var/www/html/magento2/vendor/magento/zendframework1/library/Zend/Db/Adapter/Pdo/Abstract.php:144, PDOException(code: 1698): SQLSTATE[HY000] [1698] Access denied for user 'root'@'localhost' at /var/www/html/magento2/vendor/magento/zendframework1/library/Zend/Db/Adapter/Pdo/Abstract.php:128)"} []
[2022-06-26 06:33:37] main.CRITICAL: SQLSTATE[HY000] [1698] Access denied for user 'root'@'localhost' {"exception":"[object] (Zend_Db_Adapter_Exception(code: 1698): SQLSTATE[HY000] [1698] Access denied for user 'root'@'localhost' at /var/www/html/magento2/vendor/magento/zendframework1/library/Zend/Db/Adapter/Pdo/Abstract.php:144, PDOException(code: 1698): SQLSTATE[HY000] [1698] Access denied for user 'root'@'localhost' at /var/www/html/magento2/vendor/magento/zendframework1/library/Zend/Db/Adapter/Pdo/Abstract.php:128)"} []
[2022-06-26 06:41:42] main.CRITICAL: SQLSTATE[HY000] [1044] Access denied for user 'magento2'@'localhost' to database 'magento2' {"exception":"[object] (Zend_Db_Adapter_Exception(code: 1044): SQLSTATE[HY000] [1044] Access denied for user 'magento2'@'localhost' to database 'magento2' at /var/www/html/magento2/vendor/magento/zendframework1/library/Zend/Db/Adapter/Pdo/Abstract.php:144, PDOException(code: 1044): SQLSTATE[HY000] [1044] Access denied for user 'magento2'@'localhost' to database 'magento2' at /var/www/html/magento2/vendor/magento/zendframework1/library/Zend/Db/Adapter/Pdo/Abstract.php:128)"} []
[2022-06-26 06:41:43] main.CRITICAL: SQLSTATE[HY000] [1044] Access denied for user 'magento2'@'localhost' to database 'magento2' {"exception":"[object] (Zend_Db_Adapter_Exception(code: 1044): SQLSTATE[HY000] [1044] Access denied for user 'magento2'@'localhost' to database 'magento2' at /var/www/html/magento2/vendor/magento/zendframework1/library/Zend/Db/Adapter/Pdo/Abstract.php:144, PDOException(code: 1044): SQLSTATE[HY000] [1044] Access denied for user 'magento2'@'localhost' to database 'magento2' at /var/www/html/magento2/vendor/magento/zendframework1/library/Zend/Db/Adapter/Pdo/Abstract.php:128)"} []
